k8s/desiretask.go

```diff
@@ -13,6 +13,8 @@ import (
 const (
 	ActiveDeadlineSeconds = 900
 	stagingSourceType     = "STG"
+	parallelism           = 1
+	completions           = 3
```

Comment:
Does this mean that the job will be run 3 times? Why is this required?

Reply:
@mattysweeps this variable requires task to complete three times before it is marked successful. Why is that?

Reply:
It is an oversight from when we were trying to do the staging steps in separate pods in a job. the value should be set to `1` but since parallelism is already at `1`, the value for `completions` has no impact anyway. 

That said, the last table [here](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#job-patterns) describes the relationship between `parallelism` and `completions` in a kubernetes job. It certainly doesnt mean to have the task completed three times.